pkg/configuration/configuration.go

```diff
@@ -74,16 +73,23 @@ type TemplateRepository struct {
 	// Name is the name of this module. This should be a valid go import path
 	Name string `yaml:"name"`
 
+	// Deprecated: Use 'channel' instead, prerelease sets 'channel' to 'rc'.
 	// Prerelease is a boolean indicating whether or not to consider prerelease versions
 	Prerelease bool `yaml:"prerelease"`
 
 	// Deprecated: Use name instead
 	// URL is a full URL for a given module
 	URL string `yaml:"url,omitempty"`
 
+	// Channel is the channel to use for updates to this module
+	// Defaults to "stable"
+	Channel string `yaml:"channel,omitempty"`
+
 	// Version is a semantic version or branch of the template repository
 	// that should be downloaded if not set then the latest version is used.
-	// Note: Setting this equates to pinning the versions, this is not recommended.
+	//
+	// Version can also be a constraint as supported by the underlying resolver:
+	// !!! <TODO link to gobox/pkg/cli/updater/resolver>
```

Comment:
This TODO is known, this PR has to land first: https://github.com/getoutreach/gobox/pull/219